websites/A/AnimesWatch/presence.ts

```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+		clientId: "972647321970016316",
+	}),
+	getStrings = async () =>
+		presence.getStrings(
+			{
+				playing: "general.playing",
+				paused: "general.paused",
+			},
+			await presence.getSetting<string>("lang").catch(() => "tr")
+		),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement, strings: Awaited<ReturnType<typeof getStrings>>;
+
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
+	if (!msg) return;
+	video = msg;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "animeswatch",
+			details: "Ana Sayfada Göz Gezdiriyor",
+		},
+		{ pathname } = document.location,
+		[button, timestamp] = await Promise.all([
+			presence.getSetting<boolean>("button"),
+			presence.getSetting<boolean>("timestamp"),
+		]),
+		title = document.title.slice(0, document.title.length - 14);
+
+	switch (pathname.split("/")[1]) {
+		case "":
+			presenceData.details = "Ana Sayfada Göz Gezdiriyor";
+			break;
+		case "anime":
+			presenceData.details = "Bir animeye bakıyor";
+			presenceData.state = title;
+			break;
+		case "watch":
+			presenceData.details = "Bir anime izleme sayfasında";
+			presenceData.state = title;
+
+			if (video) {
+				presenceData.smallImageKey = video.paused ? "pause" : "play";
+				presenceData.smallImageText = video.paused
+					? (await strings).paused
+					: (await strings).playing;
+				presenceData.details = "Anime izliyor";
+				if (!video.paused && video.duration) {
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestamps(
+							Math.floor(video.currentTime),
+							Math.floor(video.duration)
+						);
+				}
+			}
+			break;
+		case "team":
+			presenceData.details = "Takım sayfasında";
+			presenceData.state = title;
+			break;
+		case "browse":
+			presenceData.details = "Bir anime arıyor";
+			presenceData.state = title;
+			break;
+		case "users":
+			presenceData.details = "Bir kullanıcıya bakıyor";
+			presenceData.state = title;
+	}
+	presenceData.startTimestamp = browsingTimestamp;
+	presenceData.buttons = [
+		{ label: "Website", url: "https://animeswatch.com" },
+		{ label: "Discord", url: "https://animeswatch.com/discord" },
+	];
```

Comment:
Both buttons here go against these two guidelines, for this reason you should remove them
![image](https://user-images.githubusercontent.com/64903135/189504533-fa9e3801-f1f7-4a47-88f6-a25f3c0c887f.png)